<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindSkipListDialog</class>
 <widget class="QDialog" name="FindSkipListDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>536</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>490</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QtAddons::Plotter" name="skip_plot" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>761</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QtAddons::ImageViewerWidget" name="skip_imageviewer" native="true">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>20</y>
     <width>251</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>62</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Dose plot</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>62</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Image</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>760</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Skip list</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="skip_edit_skiplist">
      <property name="minimumSize">
       <size>
        <width>700</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>281</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Expected number of projections</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="skip_spin_expnumproj">
      <property name="maximum">
       <number>10000</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>20</y>
     <width>421</width>
     <height>66</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="skip_button_getroi">
      <property name="text">
       <string>Dose ROI</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>x0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="skip_spin_x0">
        <property name="maximum">
         <number>10000</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>y0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="skip_spin_y0">
        <property name="maximum">
         <number>10000</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="2">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>x1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="skip_spin_x1">
        <property name="maximum">
         <number>10000</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="3">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>y1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="skip_spin_y1">
        <property name="maximum">
         <number>10000</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="skip_button_getdoselist">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>170</y>
     <width>191</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>0</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string>Get dose list</string>
   </property>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>buttonBox</zorder>
  <zorder>skip_plot</zorder>
  <zorder>skip_imageviewer</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>label_7</zorder>
  <zorder>label_8</zorder>
  <zorder>skip_button_getdoselist</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtAddons::ImageViewerWidget</class>
   <extends>QWidget</extends>
   <header location="global">imageviewerwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QtAddons::Plotter</class>
   <extends>QWidget</extends>
   <header location="global">plotter.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FindSkipListDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FindSkipListDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
